ISSUE_TEMPLATE.md

```diff
@@ -14,6 +14,21 @@ Large features typically go through three stages: [Alpha, Beta, and Stable](http
 Each stage requires various approvals from various teams.  Features require several releases
 to reach Stable.
 
+The following items have to be filled before the features freeze:
+
+- Primary contact (assignee);
+- SIG, responsible for the feature;
+- Proposal link - do the public design proposal in the community repo;
+- Reviewer(s) - recommend having 2+ reviewers (at least one from code-area OWNERS file) agreed to review. Reviewers from multiple companies preferred;
+- Approver (for LGTM, likely from SIG to which feature belongs);
+- Target - Alpha / beta / stable;
+- Timeline / status;
+
+Before the coding freeze:
+
+- Release note - a short sentence, describing the feature;
+- User documentation PR link (to http://github.com/kubernetes/kubernetes.github.io)
```

Comment:
@idvoretskyi We have at least 2 challenges:

1. How can we get docs done in time such that new features can be used on beta releases / release candidates? Require docs to be written first? Disable features that don't have docs done in time? Delay cutting releases until docs are ready?

2. How can we incentivize more people to try those beta releases / release candidates and to read their release notes, so that we have a better chance that the 1.X.0 release won't be DOA? 

Along the lines of your suggestion, we could rename 1.X.0 to the release candidate, which is what it is in reality.